ansible/templates/minigraph_device.j2

```diff
@@ -1,14 +1,12 @@
   <DeviceInfos>
     <DeviceInfo>
-      <AutoNegotiation>true</AutoNegotiation>
```

Comment:
if we disable AutoNeg here but still have AutoNeg schema in linkmetadata then AutoNeg will still come up ?

Reply:
Hi @vdahiya12  sonic does not read from this file anymore. 
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-config-engine/minigraph.py - this script reads it from link metadata.

Reply:
okay, that resolves the concern.